docs/changehistory/NextVersion.md

```diff
@@ -35,6 +35,50 @@ To adjust the limit after startup, assign to [TileAdmin.gpuMemoryLimit]($fronten
 
 This feature replaces the `@alpha` `TileAdmin.Props.mobileExpirationMemoryThreshold` option.
 
+## IModelHost and IModelApp Initialization Changes
+
+Initialization processing of iModel.js applications, and in particular the order of individual steps for frontend and backend.js classes has been complicated and vague, involving several steps that vary depending on application type and platform. This release attempts to clarify and simplify that process, while maintaining backwards compatibility. In general, if your code uses [IModelHost.startup]($backend)` and [IModelApp.startup]($frontend) for web visualization, it will continue to work without changes. However, for native (desktop and mobile) apps, some refactoring may be necessary. See [IModelHost documentation]($docs/learning/backend/IModelHost.md) for appropriate backend initialization, and [IModelApp documentation]($docs/learning/frontend/IModelApp.md) for frontend initialization.
```

Comment:
Did you mean to reference `backend.js` in the first sentence?

Reply:
Also a floating back tick after `[IModelHost.startup]($backend)`

Reply:
both fixed now